tests/store/filters/test_job_filters.py

```diff
@@ -52,3 +52,17 @@ def test_filter_jobs_by_status(job_store: MockStore):
 
     # THEN the jobs attribute status should match the original
     assert jobs[0].status == TrailblazerStatus.COMPLETED
+
+
+def test_filter_jobs_by_analysis_id(job_store: MockStore):
+    """Test return jobs by analysis id."""
+    # GIVEN a store containing jobs
+
+    # WHEN retrieving a job by analysis id
+    jobs: Query = job_store.get_query(table=Job).filter_by(analysis_id=1)
```

Comment:
Do we not want to test `filter_jobs_by_analysis_id` here? A bit unsure what this tests.

Reply:
Ah, haha did not question the copilot enough